common/djangoapps/student/roles.py

```diff
@@ -246,6 +246,21 @@ def __repr__(self):
         return '<{}>'.format(self.__class__.__name__)
 
 
+@register_access_role
+class GlobalCourseCreatorRole(OrgRole):
+    """
+    A global course creator with access to all site's courses.
```

Comment:
Choosing or words is important. Currently it looks like a global course creator user can access courses from all sites. :)
Update it.

Reply:
updated it